src/control/lib/support/log.go

```diff
@@ -435,8 +438,28 @@ func getSysNameFromQuery(configPath string, log logging.Logger) ([]string, error
 	return hostNames, nil
 }
 
+func rsyncAlternateCopy(log logging.Logger, opts ...CollectLogsParams) error {
```

Comment:
The name is a little confusing since it's explicitly not using rsync. Maybe `customCopy`?

If the structure mirrors the rsync logic, could this function also serve as a helper for the rsync copy, to avoid duplication? I.e. Just have the rsync function call into this function with appropriate opts.

Reply:
renamed. I would have to think about rearranging this code to do what you suggest. But I would like to do that in a separate PR or when a bug fix is needed.